Classes/Service/DatabaseStorageService.php

```diff
@@ -445,4 +453,51 @@ protected function getStringValue($value, int $indent = 0): string
 
         return '';
     }
+
+    /**
+     * Checks if there are entries for given storage identifier.
+     *
+     * @param string $storageIdentifier
+     * @return int
+     */
+    public function getAmountOfEntriesByStorageIdentifier(string $storageIdentifier): int
+    {
+        $query = $this->databaseStorageRepository->createQuery();
```

Comment:
I think, the query is better placed inside the Repository class with a method like `countByIdentifier(string $storageIdentifier)`.